docker/local/README.md

```diff
@@ -0,0 +1,34 @@
+# Docker Deployment
+
+This document describes how home-recipes server and telemetry infrastructure can be deployed to a local Docker instance.
+
+There will be three containers deployed:
+* **home-recipes-server** exposing ports: `5000` and `2112`
+* **home-recipes-prometheus** exposing port: `9090`
+* **home-recipes-grafana** exposing port: `3000`
+
+## Preconditions
+
+It's expected that the image **kaushikchaubal/home-recipes** for container  **home-recipes-server** already exists. Deployment script will use version `latest`.
+
+## Steps
+
+From project root build docker compose
+```
+docker-compose -f docker/local/docker-compose.yml build
```

Comment:
Interesting choice. Why did you decide to use `docker-compose` to spin this up as against creating a k8s Deployment?

Reply:
The Docker local is for testing only, you don't need the whole k8s war machine for that. Just to keep things simple.